.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.6.0
```

Comment:
Ruby 2.6.0 has `Net::HTTP` security bug. We should wait until 2.6.1 release with fix

Reply:
Now we can use ruby 2.6.1. https://www.ruby-lang.org/en/news/2019/01/30/ruby-2-6-1-released/

Reply:
yes. For sure! Will update request today at the evening! =)